### ISSUE TITLE
toString methods

### ISSUE THREAD

**Issue by ZeroOne3010** [koraktor/steam-condenser#235]
In the Java version, please add proper toString methods to some key classes (such as GameAchievement, SteamGame and GameStats) in order to facilitate testing and prototyping.

**Comment by ZeroOne3010** [comment]
I've created pull request number 10 for this one.

**Comment by gcoelho** [comment]
@koraktor  Is this issue closed? Need any additional help here?

**Comment by koraktor** [comment]
No, this is still open, although koraktor/steam-condenser-java#10 has been merged, but there are plenty of other classes that could use some improvement here.

And additionally, the other implementations could use the same enhancements.

**Comment by gcoelho** [comment]
Ok, I will work on this issue. Any classes you want me to take a look first?

**Comment by koraktor** [comment]
Well, it's would be most useful to have it for the "front-end" classes, like the servers, `GameLeaderboard` and inventories.

**Comment by gcoelho** [comment]
@koraktor What code format standard do you use? I am asking to use the same while adding this methods. If possible, can you share this config file to be imported?

**Comment by koraktor** [comment]
@gcoelho There's no `codystyle.xml` or similar, but there's a [`CONTRIBUTING`](https://github.com/koraktor/steam-condenser-java/blob/master/CONTRIBUTING.md) file for each implementation and you should follow the conventions in there.

**Comment by gcoelho** [comment]
Ok, thanks! :+1: 

I just did  a commit for GameLeaderboard and Servers. Inventories classes don´t have a test class so I will create some tests too on a future commit.

If you want to take a look: gcoelho/steam-condenser-java@d8379f558d9d27a1a99735dcd70050939a18f0b3

I wiil keep working on this issue to include Inventories.
